src/ReactiveDomain/ReactiveDomain/ReactiveDomain.csproj

```diff
@@ -1,368 +1,39 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{AF28A6F8-0044-4CFA-A6EB-6000797223A0}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>ReactiveDomain</RootNamespace>
-    <AssemblyName>ReactiveDomain</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <SccProjectName>SAK</SccProjectName>
-    <SccLocalPath>SAK</SccLocalPath>
-    <SccAuxPath>SAK</SccAuxPath>
-    <SccProvider>SAK</SccProvider>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
-    <TargetFrameworkProfile />
+    <TargetFramework>net462</TargetFramework>
+    <Platforms>x64</Platforms>
+    <Version>0.7.3.0</Version>
+    <Authors>PerkinElmer</Authors>
+    <Company>PerkinElmer</Company>
+    <Copyright>Copyright © 2014-2017 PerkinElmer, Inc.</Copyright>
+    <Description />
+    <AssemblyVersion>0.7.3.0</AssemblyVersion>
```

Comment:
don't think the new format supports it, I'll double check. Also want to make sure to bump past  version 0.7.2.0

Reply:
The current version internally to PKI is 0.7.6541.31142. It would be really odd for the new format to not support that. If it doesn't, then let's keep that info in the classic AssemblyInfo files by specifying
`<PropertyGroup>`
`   <GenerateAssemblyInfo>false</GenerateAssemblyInfo>`
`</PropertyGroup>`
in the csproj file.

Reply:
`<Deterministic>False</Deterministic>` allows you to specify a version numbers as, e.g. `0.7.*`